js/adapt-contrib-matching.js

```diff
@@ -98,26 +98,33 @@ define([
         },
 
         canSubmit: function() {
-
             var canSubmit = true;
 
-            $('.matching-select option:selected', this.el).each(_.bind(function(index, element) {
-
-                var $element = $(element);
-
-                if ($element.index() === 0) {
+            $('.matching-select option:selected', this.el).each(function(index, element) {
```

Comment:
I know this selector was already present but it's not too readable or well performing. Could we update to `this.$('option').filter(':selected')`?

See also line 113.